julia/LibCEED.jl/docs/src/CeedVector.md

```diff
@@ -15,4 +15,7 @@ witharray_read
 setarray!
 syncarray!
 takearray!
+scale!
+LinearAlgebra.axpy!(a::CeedScalar, x::CeedVector, y::CeedVector)
```

Comment:
Do we want to make this `LinearAlgebra.axpy!` or `LibCEED.axpy!`?

Reply:
`LinearAlgebra` (which is in the standard library) defines `axpy!`, so it makes sense for LibCEED to extend that definition for this case (with dispatch based on the types). The other alternative would cause a function name conflict.

This line just adds the function to the documentation.

This is similar to what is done already for `norm`, `det`, etc.

Reply:
Will our axpy still work without them importing LinearAlgebra though?

Reply:
> Will our axpy still work without them importing LinearAlgebra though?

Yes, it should still work (it's exported, so it can be accessed as either `axpy!` or `LibCEED.axpy!`).